test/sslapitest.c

```diff
@@ -1967,12 +1967,12 @@ static const char *servalpn;
 static int alpn_select_cb (SSL *ssl, const unsigned char **out, unsigned char *outlen,
                     const unsigned char *in, unsigned int inlen, void *arg)
```

Comment:
While you are at it, fix width and indentation here.

Reply:
Sure, fixed